### ISSUE TITLE
Cannot create new pages in WordPress 4.8

### ISSUE THREAD

**Issue by fahaadsheikh** [fpcorso/quiz_master_next#641]
Please provide the following information when creating your issues:

## General description
I am unable to create a quiz page

## Expected behavior
When I click on create a page a page should open that allows me to enter questions on a page.

## Actual behavior
When I click on create page nothing happens and I get an error in the browser console. Attached screenshot of the error.

## Steps to reproduce the behavior
Make a quiz and click on create page
![settings muqabla wordpress - google chrome 2018-03-13 11 01 19](https://user-images.githubusercontent.com/28987119/37325078-1c0f280c-26ae-11e8-87d0-7fd0833fcefa.png)

## Feature request
If you are reporting a bug, delete this section. If you are requesting a feature, please explain what you would like to have in the plugin. Please include examples of other systems/software that may have similar features as well as the exact way you would use this feature.

**Comment by fahaadsheikh** [comment]
This was solved once I deleted the old version and updated to the version but when I did I lost all my previous quizzes

**Comment by fpcorso** [comment]
We have had a second user report this issue. Still looking into the cause of this. Trying to get access to a user's site who is encountering this error to debug further.

**Comment by fpcorso** [comment]
Ah! I finally figured out this error message! This message is shown if a user is using an older version of WordPress. I was testing in an older site using 4.8.1 and encountered this message and was able to replicate in new test environments running that older version. I did a little research and found that the wpApiSettings variable I am using here was added in 4.9 which is now 6 months old.

So, the recommended solution will be to upgrade to a recent version of WordPress. However, since about 10% of users are using 4.8, we will probably have to find an alternative to this variable for the short term.
